### ISSUE TITLE
Should we use PlatformIO?

### ISSUE THREAD

**Issue by fishnsotong** [ocean-x/erikson-1#5]
I need a way to compile Arduino sketches from the Raspberry Pi command line without opening up the IDE. I think this is a good solution. Is everyone open to using this for our project?

http://platformio.org

Talking about software, it's really quite behind hardware now. We still have yet to set up the Raspberry Pi, and up till now, all the software on GitHub is all testing and not much code that we can actually release to be used.

**Comment by IvanFengJK** [comment]
Do we set up an account?

**Comment by fishnsotong** [comment]
I don't think we need to, I just need some quiet time tomorrow to set up the Raspberry Pi and focus on code while those that can do hardware, focus on hardware. 

Best wishes,
Wayne Yeo
